modules/network-observability-includelist-example.adoc

```diff
@@ -0,0 +1,31 @@
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: CONCEPT
```

Comment:
I agree. I think I made this a concept because I am sorta punting to the linked Monitoring doc for the true procedural info, ofc its linked in the Assembly. I always get torn with NetObserv docs about how much to replicate from Logging/Monitoring docs because both are entwined with Network Observability in use cases, but I can also see how its just helpful to have those steps flat out for the customer, and still keep the link if they want more general info about creating Prometheus alerts.